tests/commands/install_uninstall_test.py

```diff
@@ -810,6 +810,46 @@ def test_post_merge_integration(tempdir_factory, store):
         assert os.path.exists('post-merge.tmp')
 
 
+def test_pre_rebase_integration(tempdir_factory, store):
```

Comment:
I suspect this isn't passing due to the `.replace('pre-', '')` bit -- I want to factor that out anyway so maybe 2 birds with one stone? https://github.com/pre-commit/pre-commit/issues/2732

Reply:
Yes! I tested removing that `replace` and the test passed! I leave it as it is.

How I can help with that refactor?